### ISSUE TITLE
Neat Slider Not Functioning - "wheel" just spins  (WordPress 4 issue?)

### ISSUE THREAD

**Issue by infoexcellnet** [Energeticthemes/neat-slider#1]
I followed the directions for getting the slider to work, but it does not.  How do I determine what the problem is?  I have it set to display at http://wp.michaelshealaw.com/
A direct link to the slider is http://wp.michaelshealaw.com/neat-slider/home-page/

**Comment by lauraMJones** [comment]
Any plans to update the slider to make it compatible for Wordpress 4.0? (pretty please!)

**Comment by infoexcellnet** [comment]
Please advise what this is in reference to.

Are you referencing a website developed and/or hosted by Excell.Net?

**Comment by mnshekh** [comment]
I am getting the same problem after creating the new slide in Neat Slider.
How to fix this issue?

**Comment by geonom** [comment]
Same for me... :-1:

**Comment by Energeticthemes** [comment]
Hi,
Can you please let me know what is issue are you really facing ? What happen when you create new slide ?

**Comment by Crazy2BU** [comment]
Yes, I also just installed your plug-in and after adding a few images I attempted to preview the slider. The page opens, but there is a spinning wheel where the slider should be...and spins, and spins, and... You get the idea, no slider. Just a spinning wheel.

Your slider looks really cool, but it is not working. Perhaps you can shed some light on the subject.

Best regards,

Dan at Lightsource Arts

**Comment by nunofreittas** [comment]
Any update on this issue?
Currently with WP Version 4.3.1, and still "wheel spinning".
